test/unit/utMatrix3x3.cpp

```diff
@@ -73,9 +73,21 @@ TEST_F(utMatrix3x3Test, FromToMatrixTest) {
 
     aiVector3D from, to;
 
+    auto random_ratio = []() -> float {
+        return static_cast<float>(rand() / RAND_MAX);
```

Comment:
This does an integer division and then casts to float which doesn't do the same thing

Reply:
You are right, changing it now.